client/src/modules/home/index.tsx

```diff
@@ -22,21 +26,32 @@ type User = NonNullable<AuthContextType['user']>;
 
 const Welcome = ({ user }: { user: User }) => {
   return (
-    <Flex alignItems={'center'} justifyContent="space-between">
-      <Heading as="h1">Welcome, {getNameText(user.name)}</Heading>
-      {!user.name && (
-        <Text>
-          You can set your name on your{' '}
-          <Link
-            href="/profile"
-            textDecoration={'underline'}
-            _hover={{ textDecoration: 'none' }}
-          >
-            profile page
-          </Link>
-        </Text>
-      )}
-    </Flex>
+    <>
+      <Alert status="error">
+        <AlertIcon />
+        <AlertTitle> This is a testing site. </AlertTitle>
+        <AlertDescription>
+          Unless you are a freeCodeCamp staff member or a Chapter maintainer,
+          please do not use this site. Any data you enter will be deleted
+          periodically.
+        </AlertDescription>
+      </Alert>
```

Comment:
Sure, I like the idea, but let's tune the tone down, because we don't want the maintainers to be on high alert because of the red color as Quincy said. Although I want to look up if that's true 🤔 

here is how it looks like in the suggestion

![image](https://user-images.githubusercontent.com/88248797/201074989-e9064403-8b0b-4821-bc65-dc0c028a3943.png)


```suggestion
      <Alert status="info">
        <AlertIcon />
        <AlertDescription>
          This is a testing site for freeCodeCamp staff member or a Chapter
          maintainer, be mindful that your data will be deleted periodically.
        </AlertDescription>
      </Alert>
 ```

Reply:
We do use red occasionally, though.  'danger zone' on /settings, for example.

I do prefer your description, with a minor tweak:

> This is a testing site for freeCodeCamp staff members and Chapter maintainers. Be mindful that your data will be deleted periodically.

Does that work for you?

Reply:
yeab 👍